src/reference-manual/transforms.qmd

```diff
@@ -478,13 +478,9 @@ For the transform, Stan uses the first part of an isometric log ratio
 transform; see [@egozcue+etal:2003] for the basic definitions and
 Chapter 3 of [@filzmoser+etal:2018] for the pivot coordinate version
 used here.  Stan uses the isometric log ratio transform because it
-induces a geometry with zero correlation among the dimensions, making
-it easier for HMC to explore than simpler alternatives such as setting
-the final element to the negative sum of the first elements; see, e.g.,
-[@seyboldt:2024].
-
-
-
+results in equal variances across the values of the the constrained sum to zero vector 
+because simpler alternatives, such as setting the final element to the negative sum of the first elements; see, e.g.,
+[@seyboldt:2024], do not result in equal variances. The $N - 1$ unconstrained parameters are independent, however, the sum to zero constraint induces a negative correlation across the constrained values.
```

Comment:
I can't quite parse 481--483.   I'd recommend 

1.  rewriting so there's not a sentence with two "because"s in it.
2.  moving the text after "see, e.g., [Seyboldt 2024]" to before it and then ending on the see, e.g., part.

"sum-to-zero" should be hyphenated when used as an adjective.

Lines should be limited to 80 characters to make diffs in GitHub easy to see.

Has the updated .bib file with @seyboldt been merged already?

Reply:
I updated the language. Let me know if there's still anything confusing or grammatically incorrect.

> Has the updated .bib file with @seyboldt been merged already?

I didn't know the bib file had _not_ been updated. I didn't check to see if it was.

Reply:
The .bib entry has already been committed, so nothing to do for this PR.

```
@misc{seyboldt:2024,
  author="Seyboldt, Adrian",
  title="Add ZeroSumNormal distribution",
  note="pyro-ppl GitHub repository issue \#1751",
  year = "2024",
  url ="https://github.com/pyro-ppl/numpyro/pull/1751#issuecomment-1980569811"
}
```